src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days > 0 && days < 3) {
+    return days * 40;
```

Comment:
It would be nice to create variables for rent price(40), small(20) and big discounts(50) and use them instead of hardcoded values

Reply:
I made changes as you requested, and now it looks more useful